src/sql/base/browser/ui/taskbar/overflowActionbar.ts

```diff
@@ -192,7 +195,7 @@ export class OverflowActionBar extends ActionBar {
 
 		this._moreItemElement.appendChild(this._moreActionsElement);
 		this._actionsList.appendChild(this._moreItemElement);
-		this._items.push(undefined); // add place holder for more item element
+		this._items.push(undefined!); // add place holder for more item element @anthonydresser, im not sure how this is working, vscode indexes into this value all the time...
```

Comment:
This seems like a pretty obvious issue, as I mention in my comment, vscode indexes into these values everywhere in their code so this seems highly likely to throw and undefined error. @kisantia